### PR TITLE
Babelrc false babelrc

### PR DESCRIPTION
**Pull request by arunoda**
Also fixes https://github.com/zeit/next.js/issues/1089

It's possible to disable babelrc via a `.babelrc` config. In that case, we need to add our default preset.

With this PR we handle that case. 

This also helps us with the examples. So, we can have a `.babelrc` file in the `examples` directory with this:

```json
{
  "babelrc": false
}
```

### CODE REVIEW

**Review by timneutkens — APPROVED** [review]
Looks good. Pretty strange though that you can disable `.babelrc` using `.babelrc` haha

File: server/build/webpack.js [hunk 1]
```diff
@@ -114,16 +114,24 @@ export default async function createCompiler (dir, { dev = false, quiet = false
...
+  if (externalBabelConfig) {
     console.log(`> Using external babel configuration`)
-    console.log(`> location: "${configLocation}"`)
-    mainBabelOptions.babelrc = true
+    console.log(`> location: "${externalBabelConfig.loc}"`)
+    // It's possible to turn off babelrc support via babelrc itself.
+    // In that case, we should add our default preset.
+    // That's why we need to do this.
+    const { options } = externalBabelConfig
+    mainBabelOptions.babelrc = options.babelrc === false ? options.babelrc : true
```
Comment (nkzawa) [inline server/build/webpack.js]: `mainBabelOptions.babelrc = options.babelrc !== false` is simpler ?

Comment (arunoda) [inline server/build/webpack.js]: Yeah!

Comment (arunoda) [inline server/build/webpack.js]: Done.

**Pull request merged**